MantidQt/CustomInterfaces/CMakeLists.txt

```diff
@@ -76,20 +76,20 @@ set ( SRC_FILES
 	src/Muon/MuonAnalysisOptionTab.cpp
 	src/Muon/MuonAnalysisResultTableTab.cpp
 	src/ParseKeyValueString.cpp
+	src/Reflectometry/GenericDataProcessorPresenter.cpp
 	src/Reflectometry/MeasurementItem.cpp
-	src/Reflectometry/QReflTableModel.cpp
-	src/Reflectometry/QReflTableView.cpp
+	src/Reflectometry/QDataProcessorTableModel.cpp
+	src/Reflectometry/QDataProcessorWidget.cpp
```

Comment:
Good. Maybe as a later step we'd like to move this new `General` features outside the `Reflectometry` sub directory